src/Service/JiraService.php

```diff
@@ -124,4 +124,24 @@ public function getCurrentUserPermissions()
 
         return $result;
     }
+
+    /**
+     * Get list of allowed permissions for current user.
+     *
+     * @return array
+     */
+    public function getPermissionsList()
+    {
+        $list = [];
+        $rest_permissions = $this->getCurrentUserPermissions();
+        if (property_exists($rest_permissions, 'permissions')) {
```

Comment:
I would use `isset` here to also check that the value is not `null`. And maybe also add ` && is_array(…)` to make sure that we have an array.